packages/core-components/src/components/LogViewer/RealLogViewer.tsx

```diff
@@ -42,18 +43,26 @@ export function RealLogViewer(props: RealLogViewerProps) {
 
   const search = useLogViewerSearch(lines);
   const selection = useLogViewerSelection(lines);
+  const location = useLocation();
 
   useEffect(() => {
     if (search.resultLine !== undefined && listRef.current) {
       listRef.current.scrollToItem(search.resultLine - 1, 'center');
     }
   }, [search.resultLine]);
 
+  useEffect(() => {
+    if (location.hash) {
+      // #line-6 -> 6
+      const line = parseInt(location.hash.replace(/\D/g, ''), 10);
+      selection.setSelection(line, false);
+    }
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
I think it makes sense to have this ES lint rule here to prevent endless re-renders.